Restructure Source Maps Upload page (#5799)
getsentry/sentry-docs: src/platform-includes/sourcemaps/upload/vite/javascript.mdx

Sentry uses [releases](/product/releases/) to match the correct source maps to your events. You can use the Sentry Vite plugin to automatically create releases and upload source maps to Sentry when bundling your app.

## Installation

```npm
npm install @sentry/vite-plugin --save-dev
```

```yarn
yarn add @sentry/vite-plugin --dev
```

## Configuration

> Learn more about the configuration of the plugin using our [Sentry Vite Plugin documentation](https://github.com/getsentry/sentry-javascript-bundler-plugins/tree/main/packages/vite-plugin).

Reviewer: ```suggestion
Learn more about configuring the plugin in our [Sentry Vite Plugin documentation](https://github.com/getsentry/sentry-javascript-bundler-plugins/tree/main/packages/vite-plugin).
```

Example:

```javascript {filename:vite.config.js}
import { defineConfig } from "vite";
import sentryVitePlugin from "@sentry/vite-plugin";

export default defineConfig({
  build: {
    sourcemap: true, // Source map generation must be turned on
  },
  plugins: [

>     // Put the Sentry esbuild plugin after any other plugins

Reviewer 2: ```suggestion
    // Put the Sentry vite plugin after any other plugins
```

Reviewer: ```suggestion
    // Put the Sentry vite plugin after all other plugins, to ensure no changes will be made to files after they're uploaded
```
